modules/embedding-microshift-into-rpm-ostree-image.adoc

```diff
@@ -0,0 +1,212 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-embed-rpm-ostree.adoc
+
+[id="embedding-microshift-rpm-ostree-image_{context}"]
+= Creating a {op-system-ostree} image embedding MicroShift
+== Adding MicroShift repositories to Image Builder
+Image Builder's `osbuild-composer` backend does not inherit the build
+host's system repos in `/etc/yum.repos.d/`. Instead, it uses its own set of
+official repositories defined in the `/usr/share/osbuild-composer/repositories`
+directory. When adding MicroShift's RPM repositories to Image Builder, you
+therefore need to use the procedure described in the Image Builder documentation
+called https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/composing_installing_and_managing_rhel_for_edge_images/managing-repositories_composing-installing-managing-rhel-for-edge-images#overriding-a-system-repository-with-support-for-subscriptions_managing-repositories[Overriding a system repository with support for subscriptions].
+
+This procedure can be summarized as copying the repository definition file for
+your {op-system} version from the `/usr/share/osbuild-composer/repositories`
+directory into the override directory at `/etc/osbuild-composer/repositories`,
+appending new repository entries for the target CPU architecture (e.g. "x86_64")
+you are building for, and reloading the `osbuild-composer` service. The
+repository definition file to copy needs to correspond to your {op-system}
+version. For example, for {op-system} 8.7, it is called `rhel-87.json`.
+
+Add MicroShift's RPM repos by using the following command to override the
+repository definition (here shown for {op-system} 8.7):
+
+[source,terminal]
+----
+$ arch=$(uname -i) jq ".$arch +=
+    [(.$arch[0] + {
+        \"name\": \"rhocp-4.12-for-rhel-8-$arch-rpms\",
+        \"baseurl\": \"https://cdn.redhat.com/content/dist/layered/rhel8/$arch/rhocp/4.12/os\"}
+    )] +
+    [(.$arch[0] + {
+        \"name\": \"fast-datapath-for-rhel-8-$arch-rpms\",
+        \"baseurl\": \"https://cdn.redhat.com/content/dist/layered/rhel8/$arch/fast-datapath/os\"}
+    )]
+    " /usr/share/osbuild-composer/repositories/rhel-87.json | sudo tee /etc/osbuild-composer/repositories/rhel-87.json
+----
+
+Verify the entries have been added. They should look like the following:
```

Comment:
We'd replace "They should look like the following:" with the .Example styling. Take a look at microshift_networking/modules/microshift-ki-cni-iptables-deleted.adoc